### DISCUSSION TITLE
Getting Corrupted epoch Half Way trough.

### DISCUSSION THREAD

**Discussion by Rai1nbow** [bmaltais/kohya_ss#3423] [Q&A]
What could the issue be? Is this a sign of overtraining? 
<img width="7765" height="1228" alt="image" src="https://github.com/user-attachments/assets/83c44d5e-2392-4e86-9c8f-84894f4165a4" />

Lora Training for style

Training images: 2000, res in ranges of as low as 704,1408 to 2048,4096. Mostly high pixel count. 

FP-16
Steps:1
Batch:2
Epoch:8
MaxTrainSteps:8000
Optimizer:Prodigy
LRScheduler: Constant
OptimizerExtrArgs:decouple=True weight_decay=0.01 betas=0.9,0.999 d_coef=2 use_bias_correction=True safeguard_warmup=True
LearningRate:0.0001
MaxRes; 576,576
EnableBuckets:True
MinBucketRes:320
MaxBucketRes:1024
TxtEncoderLR:1
UnetLearningRate:1
NetworkRankDim:96
NetworkAlpha:1
MaxTokenLenght:225

[IXLAqStyle_20250903-003827.json](https://github.com/user-attachments/files/22115102/IXLAqStyle_20250903-003827.json)

I tried following the advice on the THE OTHER LoRA TRAINING RENTRY notes. 
This is my 2nd Lora training and the first didn't have this issue but i trained it for style on 200 images and default Optimizer/Unet Learning rate.

**Comment by Rai1nbow** [comment]
I came to the conclusion that the LoRA was already overtraining by Epoch 3, with artifacts at str=1. I suspect the 1000 steps per epoch betweek 4& 5 were enough to cause this. For Prodigy, use LearningRate=1 (not 0.0001). With high-quality datasets, reduce steps or use learn dampening could be a solution. I’ve gotten good style LoRAs results at ~3k steps with 512x768. Increasing res to 1024x1024 will likely push it to ~8k steps and above.

**Comment by Rai1nbow** [comment]
I will leave the comment open for now, since i wish to hear others opinion on this.
